commit f2d86a8b3050
Author: Dimitry Andric <dimitry@andric.com>
Date:   Wed Sep 13 09:43:00 2017 -0400

    Bug 1399412 - Work around clang assertion by wrapping lz4.c in an extern C block. r=froydnj
    
    MozReview-Commit-ID: LLgSbJadbyJ
    
    --HG--
    extra : amend_source : 08d7908fe6807db654411fc291bccc98496d4275
---
 mfbt/Compression.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git mfbt/Compression.cpp mfbt/Compression.cpp
index 0a94cc69e224..c675aa90af62 100644
--- mfbt/Compression.cpp
+++ mfbt/Compression.cpp
@@ -25,8 +25,12 @@ using namespace mozilla::Compression;
 
 namespace {
 
+extern "C" {
+
 #include "lz4.c"
 
+}
+
 }/* anonymous namespace */
 
 /* Our wrappers */
